infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -576,17 +577,22 @@ private void runFailureDetectorTask() {
     }
 
     /**
-     * All Layout servers have been sealed but there is no client to take this forward and fill the
-     * slot by proposing a new layout.
+     * All active Layout servers have been sealed but there is no client to take this forward and
+     * fill the slot by proposing a new layout.
      * In this case we can pass an empty set to propose the same layout again and fill the layout
      * slot to un-block the data plane operations.
      *
      * @param pollReport Report from the polling task
      * @return True if latest layout slot is vacant. Else False.
      */
     private boolean isCurrentLayoutSlotUnFilled(PollReport pollReport) {
```

Comment:
the code where this method is called probably needs some refactoring as it is hard to figure out what is going on in the code.